scene/3d/gpu_particles_3d.cpp

```diff
@@ -462,8 +601,9 @@ GPUParticles3D::GPUParticles3D() {
 	set_one_shot(false);
 	set_amount(8);
 	set_lifetime(1);
-	set_fixed_fps(0);
+	set_fixed_fps(30);
```

Comment:
What is the reason for defaulting to 30 fixed FPS? Is it for performance?

Reply:
Interpolation has been implementated, so it should look smooth regardless of rendering FPS when fixed FPS is enabled now.